### ISSUE TITLE
Control the seed of the Random Number Generator

### ISSUE THREAD

**Issue by ffevotte** [edf-hpc/verrou#3]
Requested here: https://sourceforge.net/p/valgrind/mailman/message/35871122/

> Something that would be very useful would be control of the seed of the random number generator, so that we could repeat and debug cases that gave strange results.

**Comment by ffevotte** [comment]
And also here: https://sourceforge.net/p/valgrind/mailman/message/35871131/

> It would be nice to be able to "define" the randomness here, e.g. by
providing a pseudo-random-number generator and a command line option
to provide the "seed" value. Point is that you can actually debug
issues in a deterministic&&repeatable way *IF* they happen.

**Comment by ffevotte** [comment]
This should be fixed in 7d0c871f746eec3a799

The pRNG seed can now be specified on the command-line using the `--vr-seed` option

```
valgrind --tool=verrou --rounding-mode=random --vr-seed=12345 PROG ARGS
```
